backend/config/urls.py

```diff
@@ -15,7 +15,7 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.shortcuts import redirect
+# from django.shortcuts import redirect
```

Comment:
plz delete not comment out unused imports! don't get in a habit of letting commented out unused code linger in your work.